src/js/hca-rjsf/config/form.js

```diff
@@ -17,11 +32,86 @@ const formConfig = {
       title: 'Veteran information',
       pages: {
         veteranInformation: {
-          path: 'veteran/information',
+          path: 'veteran-information/personal-information',
+          title: 'Veteran information',
+          initialData: {},
+          uiSchema: {
+            veteranFullName: _.merge(fullNameUISchema, {
+              first: {
+                'ui:errorMessages': {
+                  minLength: 'Please provide a valid name. Must be at least 2 characters.'
```

Comment:
It looks like for the first name, the minLength is 1. The last name is 2. If I put this on the `last` field, it shows up.